test/verify/check-storage-resize

```diff
@@ -159,7 +159,7 @@ class TestStorageResize(StorageCase):
         # and unmount the fs explicitly.  But then we also need to
         # check it explicitly.
         #
-        self.machine.execute(f"umount '{fs_dev}' && fsadm -y check '{fs_dev}' && fsadm -y resize '{fs_dev}' '{size}'")
+        self.machine.execute(f"umount '{fs_dev}' || true && fsadm -y check '{fs_dev}' && fsadm -y resize '{fs_dev}' '{size}'")
```

Comment:
I don't like `|| true` in tests, it makes bugs too hard to see/debug. Please rather use `(! mountpoint {fs_dev} || umount {fs_dev}) && ...` so that we don't cover up a *failed* unmount.